src/clients/jobManagerClient.ts

```diff
@@ -14,32 +14,41 @@ export class JobManagerClient extends HttpClient {
     this.incomingSyncJobType = config.get('incoming_sync_job_type');
   }
 
-  public async getSuccessNotCleanedIngestionJobs(jobType: string): Promise<IJob<IngestionParams>[]> {
-    return this.get<IJob<IngestionParams>[]>('/jobs', {
+  public async getSuccessNotCleanedIngestionJobs(jobType: string): Promise<IJob<IWithCleanDataIngestionParams>[]> {
+    return this.get<IJob<IWithCleanDataIngestionParams>[]>('/jobs', {
       isCleaned: false,
       status: 'Completed',
       type: jobType,
       shouldReturnTasks: false,
     });
   }
 
-  public async getFailedAndNotCleanedIngestionJobs(jobType: string): Promise<IJob<IngestionParams>[]> {
-    return this.get<IJob<IngestionParams>[]>('/jobs', {
+  public async getFailedAndNotCleanedIngestionJobs(jobType: string): Promise<IJob<IWithCleanDataIngestionParams>[]> {
+    return this.get<IJob<IWithCleanDataIngestionParams>[]>('/jobs', {
       isCleaned: false,
       status: 'Failed',
       type: jobType,
       shouldReturnTasks: false,
     });
   }
 
-  public async getFailedAndNotCleanedIncomingSyncJobs(): Promise<IJob<IngestionParams>[]> {
-    const failed = await this.get<IJob<IngestionParams>[]>('/jobs', {
+  public async getInProgressJobs(jobType: string): Promise<IJob<IWithCleanDataIngestionParams>[]> {
```

Comment:
think abourt uniting the get jobs methods to obe base that is being used with different parms